VIRTUAL_VACATION/vacation.pl

```diff
@@ -208,7 +208,7 @@
 
 my $db_true; # MySQL and PgSQL use different values for TRUE, and unicode support...
 if ($db_type eq 'mysql') {
-    $dbh->do('SET CHARACTER SET utf8;');
```

Comment:
Should perl be using SET NAMES ... here ?

Reply:
Possibly, works this way for me.